mobile/packages/airqo_api/lib/src/models/prediction.dart

```diff
@@ -35,8 +35,7 @@ class Prediction {
   static List<Prediction> parsePredictions(Map<String, dynamic> jsonBody) {
     final suggestions = <Prediction>[];
 
-    final jsonArray = jsonBody['predictions'];
-    for (final jsonElement in jsonArray as List<Map<String, dynamic>>) {
```

Comment:
Removed casting which is causing failure of search feature

Reply:
Thanks @NoahNsimbe